### ISSUE TITLE
[Feature Request] Add option for how many lines of a note to display in the list

### ISSUE THREAD

**Issue by CodeCubeNeo** [quillpad/quillpad#199]
Similar to the functionality in notally. Basicaly add a number input in the settings, how many (first) lines of a note to show in the note preview on home page where notes are listed

**Comment by opk12** [comment]
Related
Related
#303 - `Preview and compact preview suggestion`
#268 - `It seems more convenient to add a preview mode`

**Comment by devguy23857848467** [comment]
Yes this feature is critical for my use, to reduce clutter to next actionable task in a list only.
